packages/polyfills/package.json

```diff
@@ -6,6 +6,9 @@
   "sideEffects": true,
   "main": "./dist/src/index.js",
```

Comment:
If everything's in `dist`, why is `src` still part of this path?

Reply:
forgot to check-in this change 🤦‍♂